codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/generators/HttpProtocolTestGenerator.java

```diff
@@ -157,8 +157,18 @@ private void renderResponseTests(OperationShape operation,
                             renderStreamingParamReader(outputShape);
                         }
                     })
-                    .write("expect(output.data.to_h).to eq($L)",
-                            getRubyHashFromParams(outputShape, testCase.getParams()))
+                    .call(() -> {
+                        if (testCase.getBodyMediaType().isPresent()
```

Comment:
Should we wrap both of these in a helper?

Reply:
yeah, good call.